articles/deploying-fleet-on-render.md

```diff
@@ -97,7 +105,9 @@ Fleet is up and running, head to your public URL.
 
 You should be prompted with a setup page, where you can enter your name, email, and password. Run through those steps and you should have an empty hosts page waiting for you.
 
-You’ll find the enroll-secret after clicking “Add New Hosts”. This is a special secret the host will need to register to your Fleet instance. Once you have the enroll-secret you can use `fleetctl` to create Orbit installers, which makes installing and updating osquery super simple. [Download fleetctl](https://github.com/fleetdm/fleet/releases/tag/fleet-v4.3.0) and try the following command (Docker require) on your terminal:
+You’ll find the enroll-secret after clicking “Add New Hosts”. This is a special secret the host will need to register to your Fleet instance. Once you have the enroll-secret you can use `fleetctl` to create Orbit installers, which makes installing and updating osquery super simple.
```

Comment:
Couple things I just noticed (already existed before this PR but worth updating):
+ Button text is just "Add hosts" nowadays (& should be sentence-cased)
+ Should remove reference to Orbit since we're working on cleaning those up — maybe just "generate installers" instead of "create Orbit installers"?